src/main/scala/com/codacy/tools/scala/seed/Printer.scala

```diff
@@ -12,7 +12,7 @@ class Printer(infoStream: PrintStream = Console.out,
               errStream: PrintStream = Console.err,
               dockerEnvironment: DockerEnvironment = new DockerEnvironment()) {
 
-  def info(message: String, error: Option[Throwable] = Option.empty[Throwable]): Unit = {
+  def debug(message: String, error: Option[Throwable] = Option.empty[Throwable]): Unit = {
     if (dockerEnvironment.debug) {
```

Comment:
This method was called info, but the first thing it did, was to check if the "debug flag" was on..